### PR TITLE
[Release_v2140] Update ReleaseNotes.md

### PR DESCRIPTION
**Pull request by KodiaqQ**
### Changes

- Added v2.14.0 template;

### Reason for changes

- Upcoming release;

### Related tickets

- 157068;

#### For the contributors:
Please add your changes (as the commit to the branch) to the list according to the template and previous notes;
Do not add tests-related notes;
Provide the list of the PRs (for all your notes) in the comment for the discussion;

### CODE REVIEW

**Comment by KodiaqQ** [comment]
@alexsu52, @ljaljushkin, @l-bat, @nikita-savelyevv, @andreyanufr, @andrey-churkin, @daniil-lyakhov, @kshpv, @AlexanderDokuchaev fill the document with your changes for the upcoming release, please.

File: ReleaseNotes.md [hunk 1]
```diff
@@ -1,5 +1,55 @@
...
+- Breaking changes:
+  - ...
+- General:
+  - ...
+- Features:
+  - ...
+- Fixes:
+  - ...
+- Improvements:
+  - The `ultralytics` version has been updated to 8.3.22.
```
Comment (andrey-churkin) [inline ReleaseNotes.md:16]: https://github.com/openvinotoolkit/nncf/pull/3038

File: ReleaseNotes.md [hunk 2]
```diff
@@ -1,5 +1,55 @@
...
+  - ...
+- Deprecations/Removals:
+  - The `tensorflow_addons` has been removed from the dependencies.
```
Comment (andrey-churkin) [inline ReleaseNotes.md:19]: https://github.com/openvinotoolkit/nncf/pull/3025

**Review by andrey-churkin — APPROVED** [review]

File: ReleaseNotes.md [hunk 3]
```diff
@@ -1,5 +1,55 @@
...
+## New in Release 2.14.0
+
+Post-training Quantization:
+
+- Breaking changes:
+  - ...
+- General:
+  - ...
+- Features:
+  - (OpenVINO) Extended support of data-free and data-aware weights compression methods ([nncf.compress_weights()](docs/usage/post_training_compression/weights_compression/Usage.md#user-guide) API) with NF4 per-channel quantization, which makes compressed LLMs more accurate and faster on NPU.
```
Comment (ljaljushkin) [inline ReleaseNotes.md:12]: https://github.com/openvinotoolkit/nncf/pull/2898

File: ReleaseNotes.md [hunk 4]
```diff
@@ -1,5 +1,55 @@
...
+  - ...
+- Fixes:
+  - (OpenVINO) Fixed GPTQ weight compression method for Stable Diffusion models.
```
Comment (ljaljushkin) [inline ReleaseNotes.md:15]: https://github.com/openvinotoolkit/nncf/pull/3002

File: ReleaseNotes.md [hunk 5]
```diff
@@ -1,5 +1,60 @@
...
+
+Post-training Quantization:
+
+- Breaking changes:
+  - ...
+- General:
+  - ...
+- Features:
+  - (OpenVINO) Extended support of data-free and data-aware weights compression methods ([nncf.compress_weights()](docs/usage/post_training_compression/weights_compression/Usage.md#user-guide) API) with NF4 per-channel quantization, which makes compressed LLMs more accurate and faster on NPU.
+  - Introduced `backup_mode` optional parameter in `nncf.compress_weights()` to specify the data type for embeddings, convolutions and last linear layers during 4-bit weights compression. Available options are INT8_ASYM by default, INT8_SYM, and NONE which retains the original floating-point precision of the model weights.
```
Comment (l-bat) [inline ReleaseNotes.md:13]: https://github.com/openvinotoolkit/nncf/pull/2978

**Review by l-bat — APPROVED** [review]

**Review by daniil-lyakhov — APPROVED** [review]

File: ReleaseNotes.md [hunk 6]
```diff
@@ -1,5 +1,64 @@
...
+  - Added preview support for the optimization of models in [Torch FX](https://pytorch.org/docs/stable/fx.html) format, nncf.quantize() and nncf.compress_weights() methods. After the optimization such models can be directly executed via [torch.compile()](https://docs.openvino.ai/2024/openvino-workflow/torch-compile.html). See [int8 quantization example](https://github.com/openvinotoolkit/nncf/tree/develop/examples/post_training_quantization/torch_fx/resnet18) for more details.
+  - ...
+- Fixes:
+  - (OpenVINO) Fixed GPTQ weight compression method for Stable Diffusion models.
+  - (Torch, ONNX) Scaled dot product attention pattern quantization setup is aligned with OpenVINO.
+  - ...
+- Improvements:
+  - The `ultralytics` version has been updated to 8.3.22.
+  - Reduction in peak memory by 30-50% for data-aware weight compression with AWQ, SE, LoRA and mixed precision algorithms.
```
Comment (nikita-savelyevv) [inline ReleaseNotes.md:22]: https://github.com/openvinotoolkit/nncf/pull/3003

File: ReleaseNotes.md [hunk 7]
```diff
@@ -1,5 +1,64 @@
...
+  - Reduction in compression time by 10-20% for weight compression with AWQ algorithm.
```
Comment (nikita-savelyevv) [inline ReleaseNotes.md:23]: https://github.com/openvinotoolkit/nncf/pull/3044

**Review by nikita-savelyevv — APPROVED** [review]

**Review by andreyanufr — APPROVED** [review]

**Review by ljaljushkin — APPROVED** [review]

File: ReleaseNotes.md [hunk 8]
```diff
@@ -1,5 +1,69 @@
...
+  - Added preview support for the optimization of models in [Torch FX](https://pytorch.org/docs/stable/fx.html) format, nncf.quantize() and nncf.compress_weights() methods. After the optimization such models can be directly executed via [torch.compile()](https://docs.openvino.ai/2024/openvino-workflow/torch-compile.html). See [int8 quantization example](https://github.com/openvinotoolkit/nncf/tree/develop/examples/post_training_quantization/torch_fx/resnet18) for more details.
+  - (OpenVINO) Introduced a new option to cache and reuse statistics for the Weight Compression algorithm, reducing the time required to find optimal compression configurations. The [TinyLlama example](https://github.com/openvinotoolkit/nncf/tree/develop/examples/llm_compression/openvino/tiny_llama_find_hyperparams) has been updated to showcase this feature.
+  - Added the `quantizer_propagation_rule` parameter, providing fine-grained control over quantizer propagation. This advanced option is designed to improve accuracy for models where quantizers with different granularity could be merged to per-tensor, potentially affecting model accuracy.
```
Comment (kshpv) [inline ReleaseNotes.md:16]: https://github.com/openvinotoolkit/nncf/pull/2994/

File: ReleaseNotes.md [hunk 9]
```diff
@@ -1,5 +1,69 @@
...
+  - Introduced `backup_mode` optional parameter in `nncf.compress_weights()` to specify the data type for embeddings, convolutions and last linear layers during 4-bit weights compression. Available options are INT8_ASYM by default, INT8_SYM, and NONE which retains the original floating-point precision of the model weights.
+  - Added preview support for the optimization of models in [Torch FX](https://pytorch.org/docs/stable/fx.html) format, nncf.quantize() and nncf.compress_weights() methods. After the optimization such models can be directly executed via [torch.compile()](https://docs.openvino.ai/2024/openvino-workflow/torch-compile.html). See [int8 quantization example](https://github.com/openvinotoolkit/nncf/tree/develop/examples/post_training_quantization/torch_fx/resnet18) for more details.
+  - (OpenVINO) Introduced a new option to cache and reuse statistics for the Weight Compression algorithm, reducing the time required to find optimal compression configurations. The [TinyLlama example](https://github.com/openvinotoolkit/nncf/tree/develop/examples/llm_compression/openvino/tiny_llama_find_hyperparams) has been updated to showcase this feature.
```
Comment (kshpv) [inline ReleaseNotes.md:15]: https://github.com/openvinotoolkit/nncf/pull/3017

File: ReleaseNotes.md [hunk 10]
```diff
@@ -1,5 +1,69 @@
...
+- Fixes:
+  - Resolved an issue with redundant quantizer insertion before elementwise operations, reducing noise introduced by quantization.
+  - (OpenVINO) Fixed GPTQ weight compression method for Stable Diffusion models.
+  - (Torch, ONNX) Scaled dot product attention pattern quantization setup is aligned with OpenVINO.
+  - ...
+- Improvements:
+  - The `ultralytics` version has been updated to 8.3.22.
+  - Reduction in peak memory by 30-50% for data-aware weight compression with AWQ, SE, LoRA and mixed precision algorithms.
+  - Reduction in compression time by 10-20% for weight compression with AWQ algorithm.
+  - (ONNX) SE-block ignored pattern variant for `torchvision` mobilenet_v3 has been extended.
```
Comment (kshpv) [inline ReleaseNotes.md:27]: https://github.com/openvinotoolkit/nncf/pull/2926

File: ReleaseNotes.md [hunk 11]
```diff
@@ -1,5 +1,69 @@
...
+  - (OpenVINO) Introduced a new option to cache and reuse statistics for the Weight Compression algorithm, reducing the time required to find optimal compression configurations. The [TinyLlama example](https://github.com/openvinotoolkit/nncf/tree/develop/examples/llm_compression/openvino/tiny_llama_find_hyperparams) has been updated to showcase this feature.
+  - Added the `quantizer_propagation_rule` parameter, providing fine-grained control over quantizer propagation. This advanced option is designed to improve accuracy for models where quantizers with different granularity could be merged to per-tensor, potentially affecting model accuracy.
+  - ...
+- Fixes:
+  - Resolved an issue with redundant quantizer insertion before elementwise operations, reducing noise introduced by quantization.
```
Comment (kshpv) [inline ReleaseNotes.md:19]: https://github.com/openvinotoolkit/nncf/pull/2903

**Review by kshpv — APPROVED** [review]

File: ReleaseNotes.md [hunk 12]
```diff
@@ -1,5 +1,69 @@
...
+Post-training Quantization:
+
+- Breaking changes:
+  - ...
+- General:
+  - ...
+- Features:
+  - (OpenVINO) Extended support of data-free and data-aware weights compression methods ([nncf.compress_weights()](docs/usage/post_training_compression/weights_compression/Usage.md#user-guide) API) with NF4 per-channel quantization, which makes compressed LLMs more accurate and faster on NPU.
+  - Introduced `backup_mode` optional parameter in `nncf.compress_weights()` to specify the data type for embeddings, convolutions and last linear layers during 4-bit weights compression. Available options are INT8_ASYM by default, INT8_SYM, and NONE which retains the original floating-point precision of the model weights.
+  - Added preview support for the optimization of models in [Torch FX](https://pytorch.org/docs/stable/fx.html) format, nncf.quantize() and nncf.compress_weights() methods. After the optimization such models can be directly executed via [torch.compile()](https://docs.openvino.ai/2024/openvino-workflow/torch-compile.html). See [int8 quantization example](https://github.com/openvinotoolkit/nncf/tree/develop/examples/post_training_quantization/torch_fx/resnet18) for more details.
```
Comment (alexsu52) [inline ReleaseNotes.md:14]:
```suggestion
 (Experimental: Torch FX) Added experemental support for quantizazation and weights compression of [Torch FX](https://pytorch.org/docs/stable/fx.html) models. The compressed models can be directly executed via [torch.compile(compressed_model, backend="openvino")](https://docs.openvino.ai/2024/openvino-workflow/torch-compile.html), see [int8 quantization example](https://github.com/openvinotoolkit/nncf/tree/develop/examples/post_training_quantization/torch_fx/resnet18). The list of supported features:
    - INT8 quantization with SmoothQuant, MinMax, Fast Bias Correction and Bias Correction algorihtms via nncf.quantize().
    - Data free INT8 and INT4 weights compression with nncf.compress_weights().
    - Data free mixed-presision data weights compression with nncf.compress_weights(). "ratio" parameter is specified the percent of the rest layers compressed to 4-bit, e.g. ratio=0.9 means 90% of layers compressed to the corresponding 4-bit data type and the rest to a `backup_mode`.
```
@daniil-lyakhov, @anzr299 I suggest to extend this bullet the following way.

File: ReleaseNotes.md [hunk 13]
```diff
@@ -1,5 +1,69 @@
...
+  - (OpenVINO) Fixed GPTQ weight compression method for Stable Diffusion models.
+  - (Torch, ONNX) Scaled dot product attention pattern quantization setup is aligned with OpenVINO.
+  - ...
+- Improvements:
+  - The `ultralytics` version has been updated to 8.3.22.
+  - Reduction in peak memory by 30-50% for data-aware weight compression with AWQ, SE, LoRA and mixed precision algorithms.
+  - Reduction in compression time by 10-20% for weight compression with AWQ algorithm.
+  - (ONNX) SE-block ignored pattern variant for `torchvision` mobilenet_v3 has been extended.
+  - ...
+- Tutorials:
```
Comment (alexsu52) [inline ReleaseNotes.md:29]: @anzr299, please, add link to the SD3 tutorial.

File: ReleaseNotes.md (hunk 12 above)
Comment (KodiaqQ) [reply ReleaseNotes.md:14]: @alexsu52, @daniil-lyakhov, @anzr299, please consider to make a decision about this ASAP.

**Review by AlexanderDokuchaev — APPROVED** [review]

**Review by alexsu52 — APPROVED** [review]

File: ReleaseNotes.md [hunk 14]
```diff
@@ -1,5 +1,56 @@
...
+  - (ONNX) SE-block ignored pattern variant for `torchvision` mobilenet_v3 has been extended.
+- Tutorials:
+  - [Post-Training Optimization of Llama-3.2-11B-Vision Model](https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/mllama-3.2/mllama-3.2.ipynb)
+  - [Post-Training Optimization of YOLOv11 Model](https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/yolov11-optimization/yolov11-object-detection.ipynb)
+  - [Post-Training Optimization of Whisper in Automatic speech recognition with OpenVINO Generate API](https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/whisper-asr-genai/whisper-asr-genai.ipynb)
+  - [Post-Training Optimization of Pixtral Model](https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/pixtral/pixtral.ipynb)
+  - [Post-Training Optimization of LLM ReAct Agent Model](https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/llm-agent-react/llm-agent-react.ipynb)
+  - [Post-Training Optimization of CatVTON Model](https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/catvton/catvton.ipynb)
```
Comment (anzr299) [inline ReleaseNotes.md:37]:
```suggestion
  - [Post-Training Optimization of CatVTON Model](https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/catvton/catvton.ipynb)
```
- [Post-Training Optimization of CatVTON Model](https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/catvton/catvton.ipynb)
- [Post-Training Optimization of Stable Diffusion v3 Model in Torch FX Representation](https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/stable-diffusion-v3/stable-diffusion-v3-torch-fx.ipynb)

**Review by MaximProshin — APPROVED** [review]

**Pull request merged**